openindiana borg create --content-from-command return code

These tests are failing on openindiana:

```
self = <borg.testsuite.archiver.ArchiverTestCase testMethod=test_create_content_from_command_with_failed_command>

    def test_create_content_from_command_with_failed_command(self):
        self.cmd('init', '--encryption=repokey', self.repository_location)
        output = self.cmd('create', '--content-from-command', self.repository_location + '::test',
                          '--', 'false', '--arg-passed-to-false', exit_code=2)
>       assert output.endswith("Command 'false' exited with status 1\n")
E       AssertionError: assert False
E        +  where False = <built-in method endswith of str object at 0xfffffd7feb435ea0>("Command 'false' exited with status 1\n")
E        +    where <built-in method endswith of str object at 0xfffffd7feb435ea0> = "Command 'false' exited with status 255\n".endswith

src/borg/testsuite/archiver.py:1064: AssertionError
_________________________________________________ RemoteArchiverTestCase.test_create_content_from_command_with_failed_command _________________________________________________

self = <borg.testsuite.archiver.RemoteArchiverTestCase testMethod=test_create_content_from_command_with_failed_command>

    def test_create_content_from_command_with_failed_command(self):
        self.cmd('init', '--encryption=repokey', self.repository_location)
        output = self.cmd('create', '--content-from-command', self.repository_location + '::test',
                          '--', 'false', '--arg-passed-to-false', exit_code=2)
>       assert output.endswith("Command 'false' exited with status 1\n")
E       AssertionError: assert False
E        +  where False = <built-in method endswith of str object at 0xfffffd7feac83d40>("Command 'false' exited with status 1\n")
E        +    where <built-in method endswith of str object at 0xfffffd7feac83d40> = "Command 'false' exited with status 255\n".endswith

src/borg/testsuite/archiver.py:1064: AssertionError
```

--- a/src/borg/testsuite/archiver.py
+++ b/src/borg/testsuite/archiver.py
@@ -1060,8 +1060,8 @@ class ArchiverTestCase(ArchiverTestCaseBase):
     def test_create_content_from_command_with_failed_command(self):
         self.cmd('init', '--encryption=repokey', self.repository_location)
         output = self.cmd('create', '--content-from-command', self.repository_location + '::test',
-                          '--', 'false', '--arg-passed-to-false', exit_code=2)
-        assert output.endswith("Command 'false' exited with status 1\n")
+                          '--', 'sh', '-c', 'exit 73;', exit_code=2)
+        assert output.endswith("Command 'sh' exited with status 73\n")
         archive_list = json.loads(self.cmd('list', '--json', self.repository_location))
         assert archive_list['archives'] == []
 

